html_app/model/Treatment.js

```diff
@@ -73,25 +73,34 @@ scb.Treatment = function scb_Treatment(data, context, parent) {
 	Object.defineProperty(self, 'duration', {
 		get : function() {
 			var time = parseFloat(data.duration_value);
-			//var days = Math.floor(time / 86400);
 			var days = Math.floor((time % 604800) / 86400);
-
 			var hours = Math.floor((time % 86400) / 3600);
-			var minutes = Math.round((time % 3600) / 60);
-			
+			var minutes = Math.floor((time % 3600) / 60);
 			var months = Math.floor(time /2592000);
-			
 			var weeks = Math.floor((time % 2592000) / 604800);
-			var now = (time < 60 );
+			var now = (time < 30 );
```

Comment:
The deleted line says "any times under 60 seconds is now", meaning 0 duration time.But in this experiment there are 30 sec treatment times. In the code above I added seconds variable, that takes values of durations that are shorter than one minute.